README-Hindi.md

```diff
@@ -0,0 +1,183 @@
+| यूनानी (Ελληνικά)                            | [sofronas](https://github.com/sofronas)                                                      | UI                                   |
+| इंडोनेशियाई (Bahasa Indonesia)               | [MarvinZhong](https://github.com/MarvinZhong)                                                | UI                                   |
+| इतालवी (Italiano)                          | [itsEmax64](https://github.com/itsEmax64)                                                    | UI                                   |
+| जापानी (日本語)                           | [NattyanTV](https://github.com/nattyan-tv)                                                   | UI                                   |
+| कोरियाई (한국어)                             | [jhk1090](https://github.com/jhk1090)                                                        | UI and [README](./README-Korean.md)  |
+| फारसी (فارسی)                             | [DrunkLeen](https://github.com/drunkleen), [Ar.dst](https://github.com/Ar-dst)               | UI and [README](./README-Persian.md) |
+| पोलिश (Polski)                             | [Akuczaku](https://github.com/Akuczaku)                                                      | UI                                   |
+| रूसी (Русский)                           | Oleg                                                                                         | UI                                   |
+| सर्बियाई                                     | [rina](https://github.com/sweatshirts)                                                       | UI                                   |
+| स्पेनी (Español)                           | [enriiquee](https://github.com/enriiquee)                                                    | UI                                   |
+| स्पेनी लैटिन (Español Latam)               | [Matyrela](https://github.com/Matyrela)                                                      | UI                                   |
+| थाई (ภาษาไทย)                              | [teerut26](https://github.com/teerut26)                                                      | UI (partial)                         |
+| तुर्की (Türkçe)                            | [mcagriaksoy](https://github.com/mcagriaksoy)                                                | UI and [README](./README-Turkish.md) |
+| यूक्रेनी (Українська)                      | [AndrejGorodnij](https://github.com/AndrejGorodnij)                                          | UI                                   |
+| वियतनामी (Tiếng Việt)                     | [7777Hecker](https://github.com/7777Hecker)                                                  | UI                                   |
+
+> क्या आप किसी अन्य भाषा के लिए अनुवाद जोड़ना चाहते हैं? CONTRIBUTING.md में ["अनुवाद जोड़ें या अद्यतन करें" अनुभाग](./CONTRIBUTING.md#add-or-update-a-translation) का पालन करें।
```

Comment:
The translation section can be left out of translated readmes

Reply:
Alright, I'll delete it.